packages/dev/core/src/assetContainer.ts

```diff
@@ -15,6 +15,9 @@ import type { Node } from "./node";
 import type { Observer } from "./Misc/observable";
 import type { ThinEngine } from "./Engines/thinEngine";
 import { InstancedMesh } from "./Meshes/instancedMesh";
+import { Light } from "./Lights";
+import { Camera } from "./Cameras";
+import { Tools } from "./Misc";

```

Comment:
```typescript
import { Light } from "./Lights";
import { Camera } from "./Cameras";
import { Tools } from "./Misc";
```
Shouldn't it be:
```typescript
import { Light } from "./Lights/light";
import { Camera } from "./Cameras/camera";
import { Tools } from "./Misc/tools";
```
?

Reply:
yup I think it is why the build is failing :-)

Reply:
Oh dang auto imports! Lemme fix this